packages/trpc/src/test/recipe.spec.js

```diff
@@ -0,0 +1,115 @@
+import { trpcSetup, tearDown } from "./utils";
+import { prisma } from "@recipesage/prisma";
+
+describe("Recipes", () => {
+  let user;
+  let trpc;
+
+  beforeAll(async () => {
+    ({ user, trpc } = await trpcSetup());
+  });
+
+  afterAll(() => {
+    return tearDown(user.id);
+  });
+
+  it("creates a recipe", async () => {
+    const recipe = await trpc.recipes.createRecipe.mutate({
+      title: "Sliced cucumber",
+      description: "Family recipe",
+      yield: "1 portion",
+      folder: "main",
+      activeTime: "30 sec",
+      totalTime: "3 hours",
+      source: "my grandmother",
+      url: "she doesn't have it",
+      notes: "on the fridge",
+      ingredients: "1 cucumber",
+      instructions: "slice it",
+      rating: 3,
+      labelIds: [],
+      imageIds: [],
+    });
+
+    expect(recipe.id).toMatch(
+      /^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-4[0-9a-fA-F]{3}-[89aAbB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$/,
+    );
+    expect(recipe.title).toEqual("Sliced cucumber");
```

Comment:
I'd love to see a check using prisma to test that a recipe was indeed created in the DB.